src/LiveComponent/README.md

```diff
@@ -183,11 +177,13 @@ namespace App\Components;
 // ...
 use Symfony\UX\LiveComponent\Attribute\LiveProp;
 
-class RandomNumberComponent implements LiveComponentInterface
+#[AsLiveComponent('random_number')]
+class RandomNumberComponent
 {
-    /** @LiveProp */
+    #[LiveProp]
```

Comment:
Out of scope for this PR, but a general question: Why is it called `LiveProp` instead of `LiveProperty`?

Reply:
Because, conceptually, these have some similarities to React props. Though, that is less true now (we had originally thought that we might have `LiveProp` and `LiveState` (the difference being that the 2nd is modifiable and the first is not). But, for simplicity (iirc), we changed just `LiveProp` with a `writable` option. 

Anyways, the "prop" comes from React props. I think it also sounds cooler ;)